### PR TITLE
fix: config overriding follow up changes

### PR DESCRIPTION
**Pull request by chloeminkyung**
fix: config overriding follow up changes

https://github.com/getdozer/dozer/pull/1736

### CODE REVIEW

File: docs/4_configuration/1_overview.md
Comment (chubei) [inline docs/4_configuration/1_overview.md:1]: I think this file is meant to show all the configuration options, so we need to think about how to properly present it

Comment (chloeminkyung) [reply docs/4_configuration/1_overview.md:1]: then I think we need to provide all the values with all the available fields without blank fields i guess but I assume it should also be working right?

Comment (chubei) [reply docs/4_configuration/1_overview.md:1]: Right. We can provide some example values. This config is not expected to run anyway

File: docs/4_configuration/1_overview.md [hunk 1]
```diff
@@ -123,7 +119,6 @@ endpoints:
               fields:
               - hvfhs_license_num
               - trip_miles
-       skip_default:  # Whether to skip the default index configuration for this endpoint.
```
Comment (chubei) [inline docs/4_configuration/1_overview.md:126]: Just noticed this description is wrong

**Pull request closed without merging**